### PR TITLE
Rename BOM and Work order fields

### PR DESCRIPTION
**Pull request by rohitwaghchaure**
1. Rename field allow_transfer_for_manufacture to include_item_in_manufacturing in BOM, Work Order
1. Removed field Transfer Material Against Job Card from BOM and Work Order and added select field Transfer Material Against with option as Work Order/Job Card

### CODE REVIEW

File: erpnext/manufacturing/doctype/job_card/job_card.py [hunk 1]
```diff
@@ -131,15 +134,28 @@ def set_transferred_qty(self):
...
 
-		self.set_status()
+		self.set_status(update_status)
+
+	def set_status(self, update_status=False):
+		self.status = {
+			"0": "Open",
+			"1": "Submitted",
+			"2": "Cancelled"
+		}[cstr(self.docstatus or 0)]
```
Comment (Alchez) [inline erpnext/manufacturing/doctype/job_card/job_card.py]: `cstr` seems unnecessary. Just use integers as the keys?

**Review by Alchez — CHANGES_REQUESTED** [review]

**Comment by rmehta** [comment]
Patch breaks

**Comment by rohitwaghchaure** [comment]
Fixed!

**Comment by rmehta** [comment]
Merging for now, but next time #NoTestNoMerge

**Pull request merged**